go.mod

```diff
@@ -0,0 +1,57 @@
+module github.com/docker/libkv
+
+go 1.17
+
+require (
+	github.com/coreos/etcd v3.3.25+incompatible
+	github.com/go-zookeeper/zk v1.0.2
```

Comment:
I wonder if we should decide not (yet) to use a go.mod here, as that would also mean that these dependencies will always be required (without a module, only dependencies for the packages that are actually used will be added in moby (I think?)

Reply:
Can we move on this PR.

With go1.17 and pruned module graphs, I think this is no longer a issue.